agent_api_rest/src/identity/documents/mod.rs

```diff
@@ -0,0 +1,52 @@
+use agent_identity::state::IdentityState;
+use agent_shared::config::SupportedDidMethod;
+use agent_shared::handlers::query_handler;
+use axum::{
+    extract::{Form, Path, State},
+    response::{IntoResponse, Response},
+    Json,
+};
+use hyper::StatusCode;
+use serde::{Deserialize, Serialize};
+use serde_json::json;
+use tracing::info;
+
+#[derive(Deserialize, Serialize)]
+pub struct GetDocumentsEndpoint {
+    #[serde(default)]
+    pub did_method: Option<SupportedDidMethod>,
+}
+
+pub(crate) async fn get_documents(
+    State(state): State<IdentityState>,
+    Form(GetDocumentsEndpoint { did_method }): Form<GetDocumentsEndpoint>,
+) -> Response {
+    info!("Request Params - did_method: {did_method:?}");
```

Comment:
This could be on `debug` level since it only logs what's passed in to the function and is logged by the general endpoint logger anyway.